docs/contributing/coding-standards/js.md

```diff
@@ -146,10 +144,10 @@ Prefer named exports over default exports to avoid compatibility issues with tra
 
 If you need to support older browsers, import the necessary [polyfills](/packages/govuk-frontend/src/govuk/vendor/polyfills) and they will be added to the environment when the feature is not supported.
 
-For example, if you want to polyfill `addEventListener` for IE8, import the Event polyfills.
+For example, if you want to polyfill `Element.closest()` for IE11, import the closest polyfill.
```

Comment:
Given we also won't be shipping JS for IE11 in future, would it be better to use an example for a different browser?

Safari has a couple of potential candidates. I believe we're currently intending to support Safari 11+ in future (albeit at the lowest 'passing' grade). 

- [Intersection Observers](https://caniuse.com/intersectionobserver) support was added in Safari 12.1.
- [`String.prototype.replaceAll`](https://caniuse.com/mdn-javascript_builtins_string_replaceall) support was added in Safari 13.1.
- Some features of [`Intl.NumberFormat`](https://caniuse.com/mdn-javascript_builtins_intl_numberformat_formatrange) weren't implemented until Safari 15.4.

Reply:
Yeah I picked this one as we only have `classList`, `closest` and `matches` now to choose from

They're all primarily for IE11 until we flick the `<script type="module">` switch

Until we have new polyfill candidates (or remove the document entirely) shall we keep these changes?

Reply:
Happy to keep it if you want, just conscious of it being something that'll need to be changed again in future.

Reply:
Yeah let's keep it. Until the day we have a new polyfilling approach, this remains our most up-to-date strategy

Reply:
We can pick it up as part of:

* https://github.com/alphagov/govuk-frontend/issues/2159